splice_machine/src/main/java/com/splicemachine/derby/utils/SpliceAdmin.java

```diff
@@ -2148,276 +2149,218 @@ public static void SHOW_CREATE_TABLE(String schemaName, String tableName, Result
     {
         Connection connection = getDefaultConn();
         try {
-            EngineUtils.verifyTableExists(connection, schemaName, tableName);
-
-            String getTableId = "SELECT T.TABLEID, T.TABLETYPE, T.COMPRESSION, T.DELIMITED, " +
-                    "T.ESCAPED, T.LINES, T.STORED, T.LOCATION" +
-                    " FROM SYSVW.SYSTABLESVIEW T " +
-                    "WHERE T.TABLETYPE IN ('T','E','S','V') " +
-                    "AND T.TABLENAME LIKE '" + tableName + "' AND T.SCHEMANAME = '" + schemaName + "'";
-
-            String tableId = "" ;
-            boolean firstCol = true;
+            TableDescriptor td = EngineUtils.verifyTableExists(connection, schemaName, tableName);
```

Comment:
I found some system procedures call `schemaName = EngineUtils.validateSchema(schemaName)` and `tableName = EngineUtils.validateTable(tableName)` before doing the actual logic but some don't, likely a 50/50 percent distribution.

These validations kind of normalize the names and I think it's good to call them. Most observable effect is that without calling them, we have to provide schema name and table name always in upper case. Otherwise they cannot be resolved.

Reply:
The system procedure call SHOW_CREATE_TABLE() takes quoted schemaname and tablename, so the names are expected to be in the exact form matching that in the data dictionary table. For example:
```
create table "t1" (a1 int, b1 int, c1 int);
splice> call syscs_util.show_create_table('SPLICE','t1');
DDL                                                                      
-------------------------------------------------------------------------
CREATE TABLE "SPLICE"."t1" (
"A1" INTEGER
,"B1" INTEGER
,"C1" INTEGER
) ;

1 row selected
```
For the ij command "show create table", it actually does some normalization(fill in scheme name, changing the name to upper case) in the below code in ij.jj:
```
|   <CREATE> <TABLE> ( i = caIdentifier() ( <PERIOD> i2 = caIdentifier() )?


		{
			if (i2 == null) {
				schema = null;
				tblname = i;
			} else {
				schema = i;
				tblname = i2;
			}
            if (schema == null) {
                schema = util.getSelectedSchema(theConnection);
            }
			return showCreateTable(schema, tblname);
```
However, there seems a general problem ij command issue that it cannot handle quoted names for the case-sensitive names. I've opened DB-9928 to track it.

Reply:
Got it. Thanks for the reply.